management-service/src/main/resources/application.properties

```diff
@@ -46,4 +46,21 @@ spring.mail.properties.mail.smtp.socketFactory.port=465
 spring.mail.properties.mail.smtp.socketFactory.class=javax.net.ssl.SSLSocketFactory
 
 metadata.base.url=${CLUSTER_METADATA_SERVICE_BASE_URL}
+
+spring.kafka.bootstrap-servers=${KAFKA_BROKER_URL}
+spring.kafka.consumer.group-id=management-service
+spring.kafka.security.protocol=SASL_PLAINTEXT
+spring.kafka.jaas.enabled=true
+spring.kafka.properties.sasl.mechanism=PLAIN
+spring.kafka.properties.sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required \
+  username="${KAFKA_CLIENT_USERNAME}" \
+  password="${KAFKA_CLIENT_PASSWORD}";
+
+spring.data.mongodb.host=${MANAGEMENT_SERVICE_MONGODB_HOST}
+spring.data.mongodb.port=${MANAGEMENT_SERVICE_MONGODB_PORT}
+spring.data.mongodb.username=${MANAGEMENT_SERVICE_MONGODB_USERNAME}
+spring.data.mongodb.password=${MANAGEMENT_SERVICE_MONGODB_PASSWORD}
+spring.data.mongodb.database=${MANAGEMENT_SERVICE_MONGODB_DATABASE}
+spring.data.mongodb.authentication-database=admin
+
```

Comment:
I might have missed it but how do you get broker topic names in the management service?

Reply:
They are specified in the annotation on the consumer function. Having it this way makes it hard to template the names, but I'll look further into it.